pkg/controllers/management/rkeworkerupgrader/plan.go

```diff
@@ -30,6 +30,8 @@ func (uh *upgradeHandler) nonWorkerPlan(node *v3.Node, cluster *v3.Cluster) (*v3
 	hostAddress := node.Status.NodeConfig.Address
 	hostDockerInfo := infos[hostAddress]
 
+	logrus.Tracef("getDockerInfo for node [%s] dockerInfo %#v", node.Name, hostDockerInfo.DockerRootDir)
```

Comment:
Good to know this. What does tracef do? :smiley:

Reply:
we've recently started printing out whole objects in trace than debug so debug becomes more readable https://github.com/rancher/rancher/issues/24994